src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -902,7 +902,7 @@ private function registerDebugConfiguration(array $config, ContainerBuilder $con
     {
         $loader->load('debug_prod.php');
 
-        if (class_exists(Stopwatch::class)) {
+        if ($container->isAvailable('symfony/stopwatch', Stopwatch::class)) {
```

Comment:
we definitely want to register `debug.stopwatch` even if it is a dev dependency. that's a debugging tool.